src/algebra/module/submodule.lean

```diff
@@ -204,7 +204,7 @@ by refine {to_fun := coe, ..}; simp [coe_smul]
 
 @[simp] theorem subtype_apply (x : p) : p.subtype x = x := rfl
 
-lemma subtype_eq_val : ((submodule.subtype p) : p → M) = subtype.val := rfl
+lemma subtype_eq_coe : ((submodule.subtype p) : p → M) = coe := rfl
```

Comment:
```suggestion
lemma coe_subtype : (p.subtype p : p → M) = coe := rfl
```
would match the name of this lemma for `submonoid` I think.

Reply:
Done!